### PR TITLE
feat: add copy sandbox link option on Sandbox Card

### PR DESCRIPTION
**Pull request by jyash97**
## What kind of change does this PR introduce?
Enhancement

## What is the current behavior?
We cannot get/copy the sandbox link from the dashboard. Usecase: I wanted to share a link of old codesandbox so I searched on the dashboard and had to open the sandbox to get the link.

## What is the new behavior?
Add an option to copy the link directly from Context Menu.

image

## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

Locally tested 4-5 sandboxes link by copying the link from the above option.

## Checklist

- [x] Testing 
- [x] Ready to be merged
      
- [x] Added myself to contributors table

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3886/commits/9d1a58ad4f41f57a79f63f02a43dd66cf6d7e8eb is at https://pr3886.build.csb.dev/s/new.

File: packages/app/src/app/pages/Dashboard/Content/SandboxCard/index.tsx [hunk 1]
```diff
@@ -81,6 +81,18 @@ type State = {
 export const DELETE_SANDBOX_DROP_KEY = 'delete';
 export const MAKE_TEMPLATE_DROP_KEY = 'makeTemplate';
 
+const copyToClipboard = (str: string) => {
```
Comment (SaraVieira) [inline packages/app/src/app/pages/Dashboard/Content/SandboxCard/index.tsx:84]: This is awesome! But we already have this function here: 
https://github.com/codesandbox/codesandbox-client/blob/63ea50dd95384eb2bff27b33e29fce4b139020ea/packages/app/src/app/overmind/effects/browser.ts#L54

You can call it from Overmind with `effects.browser.copyToClipboard` and passing the string 

Let me know if you need any help

Comment (jyash97) [reply packages/app/src/app/pages/Dashboard/Content/SandboxCard/index.tsx:84]: Hey 👋 

Thanks for the quick review, will make this change in some time.

Comment (SaraVieira) [reply packages/app/src/app/pages/Dashboard/Content/SandboxCard/index.tsx:84]: Cool! Thank you!

Comment (jyash97) [reply packages/app/src/app/pages/Dashboard/Content/SandboxCard/index.tsx:84]: As we have not refactored the Component yet so is there a way to access `effects` in Class Components? I was thinking to access the util by `useOvermind` hook but this won't work here.

Comment (SaraVieira) [reply packages/app/src/app/pages/Dashboard/Content/SandboxCard/index.tsx:84]: You are very right!

I will merge it!

Thank you!

**Pull request merged**